CONTRIBUTE.md

```diff
@@ -0,0 +1,30 @@
+# Contribution guidelines for Open Zaak
+
+If you want to contribute to Open Zaak, there are several guidelines that must be followed.
```

Comment:
*must be* sounds pretty strong/harsh to me. Maybe change it into something like "we ask you to follow these guidelines"? or something like that